PCL_RegionGrowing/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM gcc:latest as build
+FROM python:3
+
+ARG RELISE_VERSION=1.12.1
+
+RUN apt-get update && \
+    apt-get install -y \
+      libboost-all-dev \
+      libeigen3-dev \
+      libflann-dev \
+      python3-vtk7 \
+      libusb-1.0-0-dev \
+      wget \
+      cmake \
+      python3-pip \
+
+ADD . /app/build
+
+RUN wget -qO- https://github.com/PointCloudLibrary/pcl/releases/tag/pcl-${RELISE_VERSION}.tar.gz | tar xz \
+ && cd pcl-pcl-${RELISE_VERSION} \
+ && mkdir build \
+ && cd build \
+ && cmake -DCMAKE_BUILD_TYPE=Release .. \
+ && make -j2 install \
+ && cd ../
+
+RUN mkdir build && cd build && cmake .. && cmake .. && cd ..
+
+RUN pip3 install open3d
```

Comment:
Lets move this just after python install, before the install of pcl, for the cache reasons